stackslib/src/net/mod.rs

```diff
@@ -1026,6 +1029,7 @@ pub mod NackErrorCodes {
     pub const NoSuchDB: u32 = 6;
     pub const StaleVersion: u32 = 7;
     pub const StaleView: u32 = 8;
+    pub const FutureVersion: u32 = 9;
```

Comment:
Can you add a rustdoc to `FutureVersion`?

Does it mean that the client requested a newer version than the server had? And `StaleVersion` means that the client requested an older version than the version the server had?

Reply:
Yes, correct.  You could imagine the following:

1. node A asks for an inventory V for StackerDB S from node B
2. node A asks for chunk H with version V[H] from node C
3. Node C's value of V[H] is less than that of B's (e.g. C hasn't sync'ed with B yet), so C sees a request for what it believes to be a future version of a chunk